internal/broker/bind_create.go

```diff
@@ -30,17 +30,21 @@ type BindEndpoint struct {
 	tokenRequestBindingManager broker.BindingsManager
 	gardenerBindingsManager    broker.BindingsManager
 
+	tokenExpirationSeconds int
+
 	log logrus.FieldLogger
 }
 
 type BindingParams struct {
-	TokenRequest bool `json:"token_request,omit"`
+	TokenRequest           bool `json:"token_request,omit"`
+	TokenExpirationSeconds int  `json:"token_expiration_seconds,omit"`
```

Comment:
Let's remove token prefix since expiration seconds is applicable for both TokenRequest and Gardener adminkubeconfig subresource.